common/src/main/java/common/Runner.java

```diff
@@ -19,10 +19,10 @@
      * Workload Specific Parameters.
      */
     @Parameter(names = {"-a", "--app"}, description = "The application to be executed")
-    //public String application = "StreamLedger";
+    public String application = "StreamLedger";
```

Comment:
In the future, try to avoid modifying this file, we should change the configs through the command line